lucet-runtime/lucet-runtime-internals/src/error.rs

```diff
@@ -1,81 +1,76 @@
 use crate::instance::{FaultDetails, TerminationDetails};
-use failure::Fail;
+use anyhow::Error as AnyError;
+use thiserror::Error;
 
 /// Lucet runtime errors.
-#[derive(Debug, Fail)]
+#[derive(Debug, Error)]
 pub enum Error {
-    #[fail(display = "Invalid argument: {}", _0)]
+    #[error("Invalid argument: {}", _0)]
```

Comment:
This is fine too, but consider using:

```rust
#[error("Invalid argument: {0}")]
```

which I think people generally find more readable.